### ISSUE TITLE
Read image internet link from imageConverter

### ISSUE THREAD

**Issue by yueqing-wolai** [mwilliamson/mammoth.js#298]
When I use the convertImage option, there seems no way to get the image link from the image object. The image interface is below:
```
interface Image {
    contentType: string;
    read: ImageRead;
}
```
When I call read function on the image with link, it report this error message:  Could not convert URI to path. 
So is there a way to read the image link from imageConverter?

**Comment by yueqing-wolai** [comment]
> Do you have a minimal example document you can share?

Here is the file link. https://we.tl/t-QRbsslIgV1

**Comment by mwilliamson** [comment]
Thanks for sharing. The document has quite a lot of elements in, which makes it harder to work out exactly what the issue is. Would you be able to share a document that just has a single image that demonstrates the issue?

**Comment by yueqing-wolai** [comment]
Sure. Here are two images with internet link. Both of them cause mamommth.js to raise a error. 
https://we.tl/t-FvhmUsi2e9

**Comment by mwilliamson** [comment]
It looks like the issue with the first file is that the `a:blip` element doesn't have any associated image data, which should be fixed in the latest commit.

**Comment by mwilliamson** [comment]
In the second case, it looks like there's an external link:

```
<Relationship Id="rId7" Type="http://schemas.openxmlformats.org/officeDocument/2006/relationships/image" Target="https://mcusercontent.com/1a05e1e9821077a0ac6ff6b52/images/6d2a04dc-9e24-49f0-a18c-511c9724cc6c.png" TargetMode="External"/>
```

I'll have a think about the right way of handling this is: the closest match to current behaviour would be to download the file in the same way that we read embedded images, but accessing arbitrary external URLs by default doesn't feel right to me.

**Comment by yueqing-wolai** [comment]
> xternal"/>

Is there a possilbe to keep the link in img label? Something like this: <img src="external link" >

**Comment by mwilliamson** [comment]
Yes, although again there's the question of what makes sense as a default. I don't want to encourage arbitrary hotlinking, but haven't really thought enough about the use case.
